wolfcrypt/src/ecc.c

```diff
@@ -6857,6 +6933,30 @@ static int ecc_check_pubkey_order(ecc_key* key, ecc_point* pubkey, mp_int* a,
 #endif
 #endif /* !WOLFSSL_ATECC508A && !WOLFSSL_CRYPTOCELL*/
 
+int wc_ecc_get_generator(ecc_point* ecp, int curve_idx)
```

Comment:
This is not going to be needed very often.
Protect with OPENSSLALL define check.

Reply:
Protected with `OPENSSL_EXTRA` as it is used in `wolfSSL_EC_POINT_mul`.